addons/crm_mail_client_extension/controllers/main.py

```diff
@@ -6,31 +6,111 @@
 from odoo import http
 from odoo.http import request
 from odoo.tools.misc import formatLang
+from odoo.tools import html2plaintext
```

Comment:
Linting: from odoo.tools import html2plaintext should be placed before from odoo.tools.misc import formatLang (global alphabetical ordering)